swagger.v0.1.1.json

```diff
@@ -346,6 +346,59 @@
           "contentHandling": "CONVERT_TO_TEXT"
         }
       }
+    },
+    "/v0.1/request/deliveryLocationsByBarcode": {
+      "get": {
+        "summary": "Get items mapped to deliveryLocation(s) by barcode(s)",
+        "description": "Accepts multiple item barcodes, returns a special serialization of those items with an array of deliveryLocation objects.",
+        "parameters": [
+          {
+            "name": "barcodes",
+            "in": "query",
+            "required": true,
+            "type": "string",
+            "description": "Item barcode(s) to match."
+          },
+          {
+            "name": "patronId",
+            "in": "query",
+            "type": "string",
+            "description": "Patron ID, whose patron type will govern visibility of certain delivery locations."
+          }
+        ],
+        "tags": [
+          "request"
+        ],
+        "responses": {
+          "200": {
+            "description": "An array of items",
+            "schema": {
+              "$ref": "#/definitions/Item"
+            }
+          },
+          "default": {
+            "description": "Unexpected error",
+            "schema": {
+              "$ref": "#/definitions/ResourceError"
+            }
+          }
+        },
+        "x-amazon-apigateway-integration": {
+          "responses": {
+            "default": {
+              "statusCode": "200"
+            }
+          },
+          "uri": "arn:aws:apigateway:us-east-1:lambda:path/2015-03-31/functions/arn:aws:lambda:us-east-1:224280085904:function:discovery-api/invocations",
```

Comment:
Looks like this whole block of config may be mainly a convenience so that one can import a swagger doc when configuring API Gateway? I'm not sure this swagger doc continues to be any kind of authority for api-gateway config after endpoints have been set up. In any case, `uri` is described here:

http://docs.aws.amazon.com/apigateway/latest/developerguide/api-gateway-swagger-extensions-integration.html

It seems sensible to call out the lambda arn if the endpoint targets a lambda, but because this endpoint targets a public ELB, I'm removing it rather than publish that public URL publicly:

> For the HTTP integration, this is the URL of the HTTP endpoint